SparkPost/2web2ui
# SA-1439: Report Builder tabs scaffolding

### What Changed
 - Added `tab` prop to the metrics configs.
 - Changed `Tab` component to only render the tabs if there is >1.
 - Added the tabs into Report Builder
 - Changed the styling of the reportbuilder so it matches the mocks. Basically moved the 2 panels into a single panel.
 - Added state variable to determine whether or not to show the breakdown table.
 -  Fixed some older metrics stuff that was overlooked before.

### How To Test
 - Go to `/reports/summary`
 - You should see a bounce reasons tab by default. 
 - Click on the tab. Both the chart and table should be hidden
 - Remove bounce metric. The bounce reasons tab should not be visible.
 - Add rejection, click, and delayed metric. The respective tabs should be visible.
